tests/connectors/ccmixter.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = (driver, spec) => {
+
+	// Beta player
+	spec.shouldContainPlayerElement(driver, {
```

Comment:
Is there a reason to use this function to check the connector? `shouldBehaveLikeMusicSite` works fine too for me.